lib/dynflow/delayed_executors/abstract_core.rb

```diff
@@ -46,6 +46,7 @@ def with_error_handling(error_retval = nil, &block)
       def process(delayed_plans, check_time)
         processed_plan_uuids = []
         delayed_plans.each do |plan|
+          next if plan.frozen
```

Comment:
Is this really needed? If I'm reading the [delayed plan checking code](https://github.com/Dynflow/dynflow/pull/289/files#diff-59489d3bacadc3676de871dc37b984c7R127)  correctly, only plans which aren't frozen  should get loaded from the db and passed here

Reply:
That change alone didn't seem to handle it, but I will retest, its possible i didn't stop spring or something

Reply:
Yes, this appears to be required still.